test/fixtures/cookbooks/acceptance/test-suite/.acceptance/acceptance-cookbook/recipes/test.rb

```diff
@@ -0,0 +1 @@
+log File.join(ENV['PWD'], __FILE__)
```

Comment:
My solution is pretty much the same thing.  Just more code to validate that the actual recipe path is being called.  Probably overkill, so I'm good with simplifying this.